packages/gestalt/src/ComboBox.js

```diff
@@ -141,7 +141,7 @@ type Props = {
    */
   selectedOption?: OptionType,
   /**
-   * Defines the height of ComboBox: md: 40px, lg: 48px. Width is defined by parent component.
+   * Sets the size of the ComboBox: sm:'32px', md: 40px, lg: 48px. See the [size variant](https://gestalt.pinterest.systems/web/ComboBox#Size) for more details.
```

Comment:
I think the previous description was more clear? This is specifically talking about height, no?

(also, a couple of nits: no article needed before "ComboBox", and we should be consistent with quotes around the pixel values — IMO they're not needed)